src/liquid/place/bulk_create_form.html

```diff
@@ -74,7 +74,7 @@
               <tr>
                 <td>{{ row.friendly_name }}</td>
                 <td>{{ row.required }}</td>
-                <td>Property on place</td>
+                <td>Property on place{% if row.hint %}. <i>{{ row.hint }}</i>{% endif %}</td>
```

Comment:
Isn't it better to replace `Property on place` with the `hint` if it exits, it can be confusing 
![Screenshot 2024-09-13 at 10 25 22](https://github.com/user-attachments/assets/7fae065c-4f61-463f-9838-161aee822e52)